community/mutagen/APKBUILD

```diff
@@ -7,31 +7,28 @@ url="https://github.com/quodlibet/mutagen"
 arch="noarch"
 license="GPL-2.0-or-later"
 depends="python3 py-mutagen"
-makedepends="python3-dev"
-checkdepends="py3-pytest py3-setuptools py3-hypothesis"
+makedepends="py3-setuptools"
+checkdepends="py3-pytest py3-flake8 py3-hypothesis"
 subpackages="$pkgname-doc py-$pkgname:py"
 source="$pkgname-$pkgver.tar.gz::https://github.com/quodlibet/mutagen/archive/release-$pkgver.tar.gz"
 
-# we dont have all the needed deps for running the test suite yet
-options="!check"
-
 builddir="$srcdir/$pkgname-release-$pkgver"
 
 build() {
-	cd "$builddir"
-	python3 setup.py build
-}
+	# Disable failing pep8 test
+	rm -f tests/quality/test_pep8.py
```

Comment:
Is this reported upstream?

Reply:
no

Reply:
Upstream [fixed][] this already in master.

[fixed]:https://github.com/quodlibet/mutagen/commit/0ee86ef9d7e06639a388d0638732810b79998608#diff-543448b48321b1e3f4c34888542f2cf7